src/functions/template-tags/organizer.php

```diff
@@ -125,7 +125,7 @@ function tribe_get_organizer_details( $post_id = null ) {
 		}
 
 		if ( $organizer_id && $link = tribe_get_organizer_website_link() ) {
-			$details[] = '<span class="link"> <a href="' . esc_attr( $link ) . '">' . $link . '</a> </span>';
+			$details[] = '<span class="link">' . $link . '</span>';
```

Comment:
Should add a comment here that explains why `$link` isn't escaped, etc. Something like:
```
// $link is a full HTML string whose components are already escaped, so we don't need to escape again here
```